src/gatsby-node.js

```diff
@@ -70,6 +72,36 @@ exports.sourceNodes = async (
     })
   );
 
+  let newOrExistingEntries;
+
+  // Fetch only the updated data between run
+  if (lastFetched) {
+    const deltaEndpoints = endpoints.map((endpoint) => {
+      return {
+        ...endpoint,
+        queryParams: {
+          ...endpoint.queryParams,
+          // TODO
```

Comment:
what's left to do?

Reply:
I need to rework the populate and the queries...